docs/tutorials/visualizing_the_system.ipynb

```diff
@@ -0,0 +1,3760 @@
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pandas as pd\n",
+    "\n",
+    "from gettsim import set_up_policy_environment\n",
+    "from gettsim import plot_dag"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "For the visualization, we need to set up our policy environment. We do not need the parameters for that which is why the first return of the ``set_up_policy_environment`` function, the ``policy_parameters``, is stored in a variable called ``_``. This is a convention within the Python community."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "_, policy_functions = set_up_policy_environment(date=2020)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Functions inside gettsim are a little bit special. Take for example `kindergeld_m_tu` which is documented [here](https://gettsim.readthedocs.io/en/latest/functions.html#gettsim.functions.kindergeld_m_hh). The signature of the function is\n",
+    "\n",
+    "```python\n",
+    "def kindergeld_m_hh(kindergeld_m, hh_id):\n",
+    "    pass\n",
+    "```\n",
+    "\n",
+    "This functions has two arguments and none of them are parameters. Most functions require some parameters, but it is not necessary. The names of the arguments correspond to either a variable in the data provided by the user or to another function which, in turn, also relies on some arguments.\n",
+    "\n",
+    "Here, ``hh_id`` is a variable which indicates households and needs to be given by the data. ``kindergeld_m`` on the other hand is itself a function which is documented [here](https://gettsim.readthedocs.io/en/latest/functions.html#gettsim.functions.kindergeld_m_ab_1997) and [here](https://gettsim.readthedocs.io/en/latest/functions.html#gettsim.functions.kindergeld_m_bis_1996) (different versions for different time periods). By using ``kindergeld_m`` as a an argument name, gettsim knows to pass the data computed by the function ``kindergeld_m`` to ``kindergeld_m_hh``.\n",
+    "\n",
+    "This dependency relationship can be analyzed for all functions passed to gettsim and be visualized in a dag. Below you can see a plot of all variables which are directly connected to ``kindergeld_m``. The arrows point from dependencies to dependents. Each node is either a functions or a variable. By clicking on a node, you are redirected to the documentation of gettsim and if the variable is computed by a function, the function's documentation is displayed."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "\n",
+       "    <div class=\"bk-root\">\n",
+       "        <a href=\"https://bokeh.org\" target=\"_blank\" class=\"bk-logo bk-logo-small bk-logo-notebook\"></a>\n",
+       "        <span id=\"1066\">Loading BokehJS ...</span>\n",
```

Comment:
![image](https://user-images.githubusercontent.com/158456/86612514-84b7aa80-bfb0-11ea-93b6-e29ba42c5584.png)

This is how it is rendered on RTD. Expected?

Reply:
Yes, I did not find a way to remove the loading BokehJS thingy. The plots should appear though if this is the issue.

Reply:
They did appear this time 'round!